Berksfile

```diff
@@ -3,7 +3,12 @@ source 'https://supermarket.chef.io'
 metadata
 
 group :integration do
-  cookbook 'java'
-  cookbook 'minitest-handler'
-  cookbook 'stash_test', :path => 'test/cookbooks/stash_test'
+  cookbook 'java',
+    # Pending release: ark_java fails when tar is not installed
+    # See: https://github.com/agileorbit-cookbooks/java/issues/263
+    github: 'agileorbit-cookbooks/java'
```

Comment:
Yep, seems some cookbooks even add them the chefignore:
https://github.com/ramonskie/bamboo/blob/master/chefignore#L72